subprojects/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java

```diff
@@ -272,7 +272,7 @@ public void setProjectName(String projectName) {
      * @since 5.0
      */
     @Incubating
-    @Option(option = "package", description = "Set the package for source files.")
+    @Option(option = "package", description = "Set the package for source files when generating JVM projects or the namespace for source files when generating C++ projects.")
```

Comment:
+1 for adam's suggestion of ignoring this for C++ for now

Reply:
It now defaults to the project name.